### DISCUSSION TITLE
Welcome to BaseElements-Plugin Discussions!

### DISCUSSION THREAD

**Discussion by nickorr** [GoyaPtyLtd/BaseElements-Plugin#223] [Announcements]
##  Welcome !

  We’re using Discussions as a place to connect with other FileMaker developers using the BaseElements plugin. We want you to use this space to :

  * Ask questions about how to use the plugin in FileMaker Pro.
  * Share code examples.
  * Help out with documentation.
  * Ask about how to get into building your own version of the plugin.

  To get started, comment below with an intro about what you do with the BaseElements plugin.

**Comment by glitchtracker** [comment]
I use a lot Be Value function, filter out and unique are used in tons of my
scripts as they allow huge speedups dealing with SQL results
Also the write sql function thanks to BE
and BE server side external container export

Le mar. 18 juin 2024 à 10:56, Nicholas Orr ***@***.***> a
écrit :

**Comment by rbparker63** [comment]
I'm using it to automatically keep track of files that appear on a shared volume. Said filenames are then displayed in FileMaker and their size, date and thumbnails are available in FMP too.

Looks like the conversion into GitHub has remove all the function examples that existed on the old website. Will they return at some point?

Finally, I can't find any info on what's in the v5.1 beta. Is there a document somewhere?

Thank.

**Reply by nickorr** [reply]
I did try to bring across all the examples, and I've even added some new ones.  So if there's any missing or that you'd like to see, let me know.

The 5.1 beta won't have any new functionality but will be build with new versions of the various open source libraries so some patches and updates etc.
